tffm/base.py

```diff
@@ -167,7 +167,10 @@ class TFFMBaseModel(six.with_metaclass(ABCMeta, BaseEstimator)):
     """
 
 
-    def init_basemodel(self, n_epochs=100, batch_size=-1, log_dir=None, session_config=None, verbose=0, seed=None, **core_arguments):
+    def init_basemodel(self, n_epochs=100, batch_size=-1,
+                       log_dir=None,session_config=None,
+                       verbose=0, seed=None,sample_weight=None,
+                       pos_class_weight=None,**core_arguments):
```

Comment:
made weights a parameter of the model so that they can be passed around from constructor to fit method